common-rest/src/test/java/org/uniprot/api/rest/service/query/UniProtQueryProcessorTest.java

```diff
@@ -209,21 +209,16 @@ void useAndAsDefaultInNestedQuery() {
     }
 
     @Test
-    void optimisesPartOfQuery() {
-        String processedQuery = processor.processQuery("a OR P12345");
-        assertThat(processedQuery, is("a OR " + FIELD_NAME + ":P12345"));
+    void optimiseSingleTermQuery() {
+        String processedQuery = processor.processQuery("P12345");
+        assertThat(processedQuery, is(FIELD_NAME + ":P12345"));
     }
 
     @Test
-    void complexQueryWithOptimisation() {
-        String ACC = "P12345";
-        String pre = "a OR ( b AND ( +c:something AND -d:something ) AND ( ";
-        String post = " OR range:[1 TO 2] OR range:[1 TO *] OR range:[* TO 1] ) )";
-        String query = pre + ACC + post;
-        String processedQuery = processor.processQuery(query);
-        assertThat(processedQuery, is(pre + FIELD_NAME + ":" + ACC + post));
+    void doNotOptimisesPartOfQuery() {
+        String processedQuery = processor.processQuery("a OR P12345");
+        assertThat(processedQuery, is("a OR P12345"));
     }
```

Comment:
Could you please add a test case with accession in the start of the query string e.g. P12345 AND cancer